models/config.yaml

```diff
@@ -48,3 +48,6 @@ llama-[0-9]*b-4bit$:
 .*chatglm:
   mode: 'instruct'
   instruction_template: 'ChatGLM'
+.*stablelm-tuned:
+  mode: 'instruct'
+  instruction_template: 'StableLM'
```

Comment:
This should be updated to:
- recognize ALL the standard filenames of StableLM. eg `stablelm-base-alpha-7b` is one such standard filename.
- configure base as normal and the Instruct model, separately
- configure them all to 4096 token max inputs (that needs a code update I think?)

Reply:
`stablelm-base-alpha-7b` is not instruction tuned and does not support that instruction_template or instruct mode, so there's nothing needed here for it.
Are the token max inputs specified in this file? I thought they were specified somewhere else, like in presets.

Reply:
done

Reply:
need `.*` on the end to match the standard folder names

Reply:
No, I really don't think it needs that.

Reply:
The upload from Stability is <https://huggingface.co/stabilityai/stablelm-base-alpha-7b>
The folder is named `stablelm-base-alpha-7b`
If you use the UI download, git clone, or any similar method, that is the name that will be used.
The regexes here must match these standard names. Your current regex will not, as `-alpha-7b` is not accounted for.

Reply:
Incorrect. My current regex works fine on names like stablelm-base-alpha-7b. Notice how none of the other regex (eg. oasst) end with .* even though their model names all have something after the oasst. I've tested it (on tuned rather than based, but it's the same issue) and it works fine how it is. It will still be recognised even if the filename contains text after the regular expression.